components/Layout.vue

```diff
@@ -1,52 +1,50 @@
 <template>
-  <div class="relative mx-auto flex max-w-8xl justify-center sm:px-2 lg:px-8 xl:px-12">
-    <div class="hidden lg:relative lg:block lg:flex-none">
-      <div class="absolute inset-y-0 right-0 w-[50vw] bg-slate-50 dark:hidden" />
-      <div class="sticky top-[4.5rem] -ml-0.5 h-[calc(100vh-4.5rem)] overflow-y-auto py-16 pl-0.5">
-        <div class="absolute top-16 bottom-0 right-0 hidden h-12 w-px bg-gradient-to-t from-slate-800 dark:block " />
-        <div class="absolute top-28 bottom-0 right-0 hidden w-px bg-slate-800 dark:block" />
+  <div class="mx-auto max-w-7xl flex flex-col-reverse min-h-full lg:grid lg:grid-cols-5 lg:gap-8 lg:px-8">
```

Comment:
Display behaviour:
+ Screen < large - `flex-col-reverse` puts the ToC at the top of the page
+ Screen >= large: Using a flex grid to size the 3 "columns" proportionally